sysl2/sysl/parse/listener_impl.go

```diff
@@ -1053,8 +1053,9 @@ func (s *TreeShapeListener) EnterName_with_attribs(ctx *parser.Name_with_attribs
 			mergeAttrs(attrs, s.module.Apps[s.appname].Attrs)
 		}
 	}
-	s.module.Apps[s.appname].SourceContext = buildSourceContext(
-		s.filename, ctx.GetStart().GetLine(), ctx.GetStart().GetColumn()+1)
+	sc := s.sc.Get(ctx.BaseParserRuleContext)
```

Comment:
@kasunfdo Do you know why this one SourceContext adds the +1 to the col when none of the others do?